src/intl/en.json

```diff
@@ -1049,6 +1037,11 @@
         "starts": "Starts",
         "ends": "Ends",
         "ended": "Ended"
+      },
+      "legislative_tracker": {
+        "title": "How did we get here?",
+        "description": "Find all the binding outcomes from past DAO initiatives in this living document",
```

Comment:
title: How do we govern?
body: Check the latest binding Governance decisions the community approved on this living document maintained by the Facilitation Squad
CTA: Legislative Tracker